### ISSUE TITLE
How to switch board to arm mode ?

### ISSUE THREAD

**Issue by Cpasjuste** [sipeed/LicheeRV-Nano-Build#59]
Hi,

To enable the a53 arm core, do we need to remove the "RISC-V C906 / arm A53 Select" resitor ? 

Thanks.

**Comment by Taumille** [comment]
Hello, as far as I know yes, the switch can only be made through this resistor. There is three pins, VDD, CPU_SEL and GND, by default CPU_SEL is connected to VDD so the selected CPU is the RISC-V C906 but you can connect it to GND to use the Arm core, here is a part of [the schematic](http://cn.dl.sipeed.com/shareURL/LICHEE/LicheeRV_Nano/02_Schematic)

![2024_11_06-14h02m45s](https://github.com/user-attachments/assets/cd9f5270-22ed-4e93-8752-89c2a341b117)

If you want to use the SG2002 SoC with its Arm CPU, you can buy the MilkV Duo 256M which has a "Boot Switch" pin that can easily be grounded.

**Comment by Cpasjuste** [comment]
Hi @Taumille,

First, thanks for the valuable information, it's hard to find sometime for this device. 

Else I think I won't go for the soldering on those tiny pads, while I have the necessary tools I won't bother with that I guess. 

Since I was not sure of the "switch" stuff, I started digging with the "riscv side", and it's fine/cool. That said, this repo is a little "chaotic", no offense. So I did start looking around and found your patches, then your GitHub and all the work you've done with this device. I must say that I'm very impressed, while there's some work left, it's very clean. Thanks for your work mate :)

I just forked your work today, hopefully I may be able to help a little (but while I'm developing "for fun" since maybe 15 years on many various stuff, Im far from your level :)).

Et vive la France :p
